projects/laji/src/app/+project-form/results/common/result-map/result-map.component.ts

```diff
@@ -182,6 +129,62 @@ export class ResultMapComponent implements OnInit {
     this.mapOptions = {
       clickBeforeZoomAndPan: true
     };
+
+    this.gatheringCountMapData$ = of([this.getGatheringCounts$(), this.getGatheringCounts$(true)]).pipe(
```

Comment:
The value inside of `of()` is just being thrown away, so you might as well call of with no arguments.

Reply:
Goes for the other one too